pkg/features/kube_features.go

```diff
@@ -1206,38 +1206,58 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	// inherited features from generic apiserver, relisted here to get a conflict if it is changed
 	// unintentionally on either side:
 
```

Comment:
nit: if we're sweeping this anyway, sort the generic features below alphabetically with a space between each line; if we do that consistently going forward, that'll help minimize conflicts in the future when multiple gates get added, instead of concurrent PRs appending to the end and conflicting

Reply:
Can we enforce that feature gates are always sorted going forward?

Reply:
@liggitt - What do you think about the enj recommendation? Also, I believe generic features registered here are already sorted  alphabetically.

Reply:
something like this in `hack/verify-test-featuregates.sh` would work

```sh
lines=$(git grep 'genericfeatures[.].*:' -- pkg/features/kube_features.go)
sorted_lines=$(echo "$lines" | sort)
if [[ "$lines" != "$sorted_lines" ]]; then
  echo "Generic features in pkg/features/kube_features.go not sorted" >&2
  echo >&2
  echo "Expected:" >&2
  echo "$sorted_lines" >&2
  echo >&2
  echo "Got:" >&2
  echo "$lines" >&2
  rc=1
fi
```

Reply:
Implemented. I've added the `-f' to the `sort' to avoid any issues with case sensitivity.